Valeriia/Movies_database_project/src/components/Movies/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Item from "../Item";
+import { IMovie } from "../../types";
+import "./index.scss";
+
+interface IMoviesProps {
+  movies: IMovie[];
+  onClickHandler(id: number): void;
```

Comment:
try to use one pattern of naming. Cause you already have
onClick
onClickHandler
click